MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -1581,7 +1582,7 @@ static MTLVertexFormat mvkAdjustFormatVectorToSize(MTLVertexFormat format, uint3
 	if (pCreateInfo->pInputAssemblyState) {
 		plDesc.inputPrimitiveTopologyMVK = isRenderingPoints(pCreateInfo)
 												? MTLPrimitiveTopologyClassPoint
-												: mvkMTLPrimitiveTopologyClassFromVkPrimitiveTopology(pCreateInfo->pInputAssemblyState->topology);
+												: mvkMTLPrimitiveTopologyClassFromVkPrimitiveTopologyInObj(pCreateInfo->pInputAssemblyState->topology, this);
```

Comment:
I thought we had a macro that did that, like we do for the other ones...

Reply:
Thanks. You are of course correct. I got confused, forgot about the macro design, and thought we were calling the public API versions. 🤦🏻‍♂️

Fixed now.